src/typing.py

```diff
@@ -1985,20 +1987,29 @@ class Employee(NamedTuple):
         The resulting class has one extra attribute: _field_types,
         giving a dict mapping field names to types.  (The field names
         are in the _fields attribute, which is part of the namedtuple
-        API.) Backward-compatible usage::
+        API.) Alternative equivalent keyword syntax is also accepted::
```

Comment:
Can you change the earlier part of the docstring (line 1975) to emphasize that the both the PEP 526 syntax and the keyword args syntax are only supported in Python 3.6+? (I can't add github comments that far back.)